<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeaBattleWindowClass</class>
 <widget class="QDialog" name="SeaBattleWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Морской бой</string>
  </property>
  <widget class="QLabel" name="lPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваше поле:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="FieldView" name="fvPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="FieldView" name="fvEnemy">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lEnemy">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Поле противника:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bWaitConnection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ждать подключения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConnect">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIPAddress">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Введите IP-адрес</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FieldView</class>
   <extends>QFrame</extends>
   <header>FieldView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bWaitConnection</tabstop>
  <tabstop>leIPAddress</tabstop>
  <tabstop>bConnect</tabstop>
 </tabstops>
 <resources>
  <include location="SeaBattleWindow.qrc"/>
 </resources>
 <connections/>
</ui>
